### PR TITLE
Fix folder deletion

### LINKED ISSUE #13: Recursive folder deletion
#### The problem

When executing `meteor-build-client ../` in de Meteor application folder. It deletes **everything**. Looking at the code, this makes sense. But when you want to export the static files one folder higher, it's unexpected behavior and can even cause users to lose all their work.
#### Steps to reproduce
- Create a folder called `foo/` and `cd` into it.
- `meteor create app`
- `cd app`
- `meteor-build-client ../`
- You get the following error:
  
  ``` bash
  Bundling Meteor app...
  
  fs.js:612
    return binding.rmdir(pathModule._makeLong(path));
                   ^
  Error: ENOENT, no such file or directory './../'
      at Object.fs.rmdirSync (fs.js:612:18)
      at deleteFolderRecursive (/usr/local/lib/node_modules/meteor-build-client/meteor.js:48:12)
      at Object.module.exports.build (/usr/local/lib/node_modules/meteor-build-client/meteor.js:56:9)
      at /usr/local/lib/node_modules/meteor-build-client/main.js:68:20
      at null._onTimeout (/usr/local/lib/node_modules/meteor-build-client/queue.js:41:11)
      at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)
  ```
  
  And the `foo/` folder is completely empty.

In this scenario the user would lose all files inside `foo/`, including the folder where the Meteor application is located. This should be fixed.

### PR DESCRIPTION
**Pull request by stephan281094**
This PR fixes #13.

This is the first time working on a node package, so feedback is appreciated.

### CODE REVIEW

**Comment by stephan281094** [comment]
I just realized that with this current solution, the other files from the 'public' folder will not get removed..

File: meteor.js [hunk 1]
```diff
@@ -49,21 +62,41 @@ var deleteFolderRecursive = function(path) {
     }
 };
 
+var deleteBuildFiles = function(buildPath) {
+    var files = [];
+    if (fs.existsSync(buildPath)) {
+        files = fs.readdirSync(buildPath);
+        files.forEach(function (file, index) {
+            var curPath = buildPath + "/" + file;
+            if (helpers.endsWith(file, ['.css', '.html', '.map', '.js'])) {
```
Comment (frozeman) [inline meteor.js]: There can be many more other files, for example assets etc..

Comment (stephan281094) [inline meteor.js]: Yeah, you're right. But then what would be the appropriate solution? Just throwing an exception when the user tries to build files in a parent directory?

Comment (frozeman) [inline meteor.js]: Yes i think thats the best. IMO you should never put build files in a parent folder anyway.

**Comment by stephan281094** [comment]
I'm closing this due to inactivity.

**Pull request closed without merging**